cassandane/tiny-tests/JMAPCalendars/calendarevent-get-locations-geo

```diff
@@ -11,5 +11,6 @@ sub test_calendarevent_get_locations_geo
     my $event = $self->putandget_vevent($id, $ical);
     my @locations = values %{$event->{locations}};
     $self->assert_num_equals(1, scalar @locations);
-    $self->assert_str_equals("geo:37.38601,-122.08290", $locations[0]{coordinates});
+    $self->assert_matches(qr{\Ageo:37\.38601\d*,-122\.08290\d*\Z},
```

Comment:
All digits, if any, after the expected latitude/longitude values must be zero. So we should not check for `\d` but `0`.

Reply:
As far as I know, libical previously stored internally the coordinates as float.  When these floats were converted to strings, there were many digits, not necessary zero, after the decimal point.  The reason is that saving one number in float and printing that number later to a string, does not reproduce exactly the initially assigned value.

Reply:
Right, that particular change is at https://github.com/libical/libical/commit/b0aa3088b8b7aeafba3624c957e8eea3ad72b7a7#diff-1dd2e567bcf8e9ee5fbfc53404fe5907d3bab50954eaafcc0c30cbdbede810fdL1105
But in this test we know that the remaining digits must be zero, if any.

Reply:
I haven't read up on this in great detail, but from what I have read, my understanding is that there has only ever been 5 significant figures after the decimal anyway, no matter how many digits were actually returned.  In which case, the additional digits being zero-or-otherwise could only be an implementation accident, not any additional precision.  So in my mind the correct thing for the test to do is match the precise digits exactly, and ignore any additional digits.

As long as it remains true that only five of those digits are significant, then implementing the test like this protects us from future underlying implementation changes, and also provides a certain documentation of the reality that we only expect five of these digits to have any meaning.

If at some point we start getting a 6th significant figure, then we'd make a similar change, to exactly match 6 figures and disregard any noise following.

Reply:
I'm convinced. Let's merge this as is.